ActiveRecord Basics guide: correct explanation of the `updated_at` logic [ci skip]

64kramsystem opened the pull request:
It's misleanding/incorrect to state that `updated_at` is set on updates, since creation != update (and the column is actually set on creation, too).

eileencodes commented on guides/source/active_record_basics.md:
```diff
@@ -115,7 +115,7 @@ to Active Record instances:
   record is first created.
 * `updated_at` - Automatically gets set to the current date and time whenever
-  the record is updated.
+  the record is saved.
```
I think this should be "the record is created or updated" because simply calling `save!` on a record won't change the timestamp so I think "the record is saved" is also misleading.

saveriomiroddi commented:
Updated (ha...), there you go :smiley:

eileencodes commented:
@saveriomiroddi can you squash your commits into one?

64kramsystem committed:
ActiveRecord Basics guide: correct explanation of the `updated_at` logic [ci skip]

It's misleanding/incorrect to state that `updated_at` is set on updates, since creation != update (and the column is actually set on creation, too).

saveriomiroddi commented:
There you go!

eileencodes commented:
Thank you!
